bika/health/catalog/patient_catalog.py

```diff
@@ -211,3 +126,41 @@ def indexObject(obj, path):
 
 
 InitializeClass(BikaHealthCatalogPatientListing)
+
+
+# TODO: Remove BikaPatientCatalog
+# @deprecated(comment="bika.health.catalog.BikaPatientCatalog "
+#                     "is deprecated and will be removed "
+#                     "in Bika Health 3.3. Please, use "
+#                     "BikaHealthCatalogPatientListing intead")
+class BikaPatientCatalog(CatalogTool):
+
+    """ Catalog for patients
+    """
+    security = ClassSecurityInfo()
+    _properties = ({'id': 'title', 'type': 'string', 'mode': 'w'},)
+
+    title = 'Bika Patient Catalog'
+    id = 'bika_patient_catalog'
+    portal_type = meta_type = 'BikaPatientCatalog'
+    plone_tool = 1
+
+    def __init__(self):
+        ZCatalog.__init__(self, self.id)
+
+    security.declareProtected(ManagePortal, 'clearFindAndRebuild')
+
+    def clearFindAndRebuild(self):
```

Comment:
This is not longer required. `BikaPatientCatalog` should inherit from `bika.lims.catalog.bika_catalog_tool.BikaCatalogTool`. See https://github.com/naralabs/bika.lims/blob/wip/bika/lims/catalog/analysisrequest_catalog.py#L125